src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball extends ColorSupplier {
```

Comment:
Ball is not a color supplier. Why do you extend it from ColorSupplier?
```suggestion
public class Ball {
```